packages/cerebral-provider-forms/src/form.js

```diff
@@ -61,6 +61,9 @@ export default function computedForm (formValueTag) {
   return compute(
     formValueTag,
     (formValue) => {
+      if (typeof formValue !== 'object') {
```

Comment:
I think that there are valid cases where a formTag/compute may sometimes resolve to undefined or null (such as on startup). Maybe we need to handle these cases without a throw?